src/helics/cpp98/CPPLINT.cfg

```diff
@@ -0,0 +1,24 @@
+set noparent
```

Comment:
If this line gets removed it might be possible to just use `filter=-build/header_guard # this doesn't match the HELICS style for the C shared library` in this file.

Reply:
that makes things a little simpler